CryptoAnalysis/src/main/java/crypto/analysis/CrySLRulesetSelector.java

```diff
@@ -58,7 +60,7 @@ public String toString() {
 			}
 		}
 		if (ruleset.isEmpty()) {
-			throw new RuntimeException("Could not parse " + rulesetString + ". Was not able to find rulesets.");
```

Comment:
So the idea of that exception was that it breaks flow to `makeFromRuleset()`.  Currently this is not the case anymore probably will result in a really unknown program state. Either throw the new exception here or just return the empty list.

*Again I think the whole class is not really CryptoAnalysis but rather CrySL but let it stay here for a while. There is a separate Issue (and PR) for this already i think. 
*